libs/ledger-live-common/src/exchange/swap/completeExchange.ts

```diff
@@ -49,7 +50,7 @@ const completeExchange = (
     const confirmExchange = async () => {
       await withDevicePromise(deviceId, async transport => {
         const providerConfig = await getSwapProvider(provider);
-        if (providerConfig.type !== "CEX") {
+        if (providerConfig.useInExchangeApp === true) {
           throw new Error(`Unsupported provider type ${providerConfig.type}`);
         }
```

Comment:
you inverted the condition here .... I guess that's why all providers boolean where inverted too 
```suggestion
        if (providerConfig.useInExchangeApp !== true) {
          throw new Error(`Unsupported provider type ${providerConfig.type}`);
        }
```